README.md

```diff
@@ -352,14 +352,16 @@ conf.loadFile(process.env.CONFIG_FILES.split(','));
 ### config.validate([options])
 
 Validates `config` against the schema used to initialize it. All errors are
-collected and thrown at once.
+collected and thrown or displayed at once.
 
-Options: At the moment `strict` is the only available option.
-
-If the `strict` option is passed (that is `{strict: true}` is passed), any
+#### Options:
+1. `strict`: If the `strict` option is passed (that is `{allowed: 'strict'}` is passed), any
```

Comment:
`strict` is not an option. `allowed` is. `strict` is just a possible value of the `allowed` option